freemocap/__init__.py

```diff
@@ -3,249 +3,7 @@
-            sesh.dlc_fr_mar_dim = reconstruct3D.reconstruct3D(
-                sesh, sesh.dlcData_nCams_nFrames_nImgPts_XYC, confidenceThreshold=0.95
-            )
-            np.save(
-                sesh.dataArrayPath / "deepLabCut_3d.npy", sesh.dlc_fr_mar_dim
-            )  # save data to npy
-
-    else:
-        print("Skipping Run MediaPipe")
-
-    # # %% Stage Five
-    # if not stage > 5:
-    #     print('Starting Parse MediaPipe')
-    #     sesh.mediaPipeData_nCams_nFrames_nImgPts_XYC = fmc_mediapipe.parseMediaPipe(sesh)
-    # else:
-    #     print('Skipping Parse MediaPipe')
-
-
-    # # %% Stage Six
-    # if not stage > 6:
-    #     print()
-    #     print('Starting Skeleton Reconstruction')
-    #     sesh.skel_fr_mar_dim = reconstruct3D.reconstruct3D(sesh,sesh.mediaPipeData_nCams_nFrames_nImgPts_XYC, confidenceThreshold=.1)
-
-    #     path_to_skel_points = sesh.dataArrayPath/'skeleton_points.npy'
-    #     np.save(path_to_skel_points, sesh.skel_fr_mar_dim)
-    # else:
-    #     print('Skipping Skeleton Reconstruction')
-
-
-    # reupdate the config_settings with mediapipe and openpose image paths
-    #sesh.config_settings = recordingconfig.load_config_yaml(sesh.yamlPath)
-
-
-    # %% Stage Six
-    if stage <= 6:
-        print("Starting PyQT Animation")
-        # createvideo.createBodyTrackingVideos(sesh)
-        displayVid = 1
-        # if displayVid = 0, will show the synced videos
-        # if displayVid = 1, will show the openPosed videos
-        playWin = PlayerDockedWindow(sesh, displayVid)
-        playWin.animate()
-
-    # %% Stage Seven
-    if stage <= 7:
-        print("Starting Skeleton Plotting")
-        playskeleton.ReplaySkeleton_matplotlib(
-            sesh,
-            vidType=1,
-            startFrame=40,
-            azimuth=-90,
-            elevation=-80,
-            useOpenPose=sesh.useOpenPose,
-            useMediaPipe=sesh.useMediaPipe,
-            useDLC=sesh.useDLC,
-        )
-        # fmc_pyqtgraph.PlaySkeleton(
-        #                             sesh,
-        #                             vidType=1,
-        #                             startFrame=40,
-        #                             azimuth=-90,
-        #                             elevation=-80,
-        #                             useOpenPose=useOpenPose,
-        #                             useMediaPipe=useMediaPipe,
-        #                             useDLC=useDLC)
-
-        # playSkel = PlaySkeleton(sesh)
-        # playSkel.animate()
-        # playWin =PlayerDockedWindow(sesh)
-        # playWin.animate()
-
-    else:
-        print("Skipping Skeleton Plotting")
-
-
-    # %% Stage Eight
-    if stage <= 8:
-        print()
-        print("Starting Video Creation")
-        createvideo.createVideo(sesh)
-    else:
-        print("Skipping Video Creation")
-
-
-    f = 9
+def RunMe(sessionID,stage,useOpenPose,useMediaPipe,useDLC,debug):
```

Comment:
@aaroncherian - btw, you can set default values for functions like ths - 

def RunMe(
sessionID=None,
stage=1,
useOpenPose=True,
useMediaPipe=False,
useDLC=True,
debug=False
)

Reply:
if the user specifies any of these inputs, it will respect it, but if they leave it blank (i.e. `Run()`) it uses the default values

Reply:
got it - I added the default values to both the RunMe (from init) and Run (from fmc_runme), so that way whether you run freemocap.RunMe() or the if name = main 'Run()', you can use default values